src/cli.ts

```diff
@@ -74,9 +75,13 @@ async function run(verb: string): Promise<boolean> {
     yes: cli.flags.yes || cli.flags.y || false,
     logger: logger
   };
+  // See: https://github.com/google/ts-style/issues/48
```

Comment:
nit: it would be better to explain the reason why we are doing this here in a comment. You can additionally link to the issue, but the gist of the intention should be clear here.